src/components/landing/Information.jsx

```diff
@@ -0,0 +1,39 @@
+import Image from "next/image";
+import image from "/public/transparent_logo_1.png";
+// import { Hubballi } from "next/font/google";
+
+// const hubballi = Hubballi({
+//   subsets: ["latin"],
+//   weight: ["400"],
+// });
+
+const Information = () => {
+  return (
+    <div style={{ display: "flex", alignItems: "center" }}>
```

Comment:
use tailwind only, check docs. for example: for the alignItems look into the align category